lib/dealership.rb

```diff
@@ -0,0 +1,42 @@
+class Dealership
+
+  attr_reader :inventory
+  def initialize(name, address)
+  @name = name
+  @address = address
+  @inventory = []
+  @total_value = 0
+  end
+
+  def inventory_count
+    @inventory.count
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    @inventory.count > 1
+  end
```

Comment:
This should return `true` if `inventory.count == 1` also so we should be checking that it's greater than 0

```suggestion
  def has_inventory?
    @inventory.count > 0
  end
```